dockerfiles/lib/typescript/src/typings/requirejs/require.d.ts

```diff
@@ -0,0 +1,397 @@
+// Type definitions for RequireJS 2.1.20
```

Comment:
@skabashnyuk I'm not sure we need CQ as it's to check the compilation, not the runtime
Or what I could do is to remote .ts file and let it be downloaded when compiling project